common/supplemental/likelySubtags.xml

```diff
@@ -3314,8 +3314,6 @@ not be patched by hand, as any changes made in that fashion may be lost.
 		<!--{ ?; Arabic; India } => { Urdu; Arabic; India }-->
 		<likelySubtag from="und_Arab_KH" to="cja_Arab_KH"/>
 		<!--{ ?; Arabic; Cambodia } => { Western Cham; Arabic; Cambodia }-->
-		<likelySubtag from="und_Arab_MM" to="rhg_Arab_MM"/>
-		<!--{ ?; Arabic; Myanmar (Burma) } => { Rohingya; Arabic; Myanmar (Burma) }-->
```

Comment:
@pedberg-icu GenerateMaximalLocales erased `<likelySubtag from="und_Arab_MM" to="rhg_Arab_MM"/>` from likely subtags.

Reply:
FYI -- I saw a couple of such deletions when I ran that for PR #1291 but I only retained changes relevant to the new scripts I was adding.

Reply:
> FYI -- I saw a couple of such deletions when I ran that for PR #1291 but I only retained changes relevant to the new scripts I was adding.

yes, i will revert.   @pedberg-icu can you figure out what needs to change so that Rohingya generated data is correct?

Reply:
> @pedberg-icu can you figure out what needs to change so that Rohingya generated data is correct?

Steven, I suggest that you back out the Rohingya changes here. Leaving the likelySubtags in the brittle state that they are in already (out of sync with the generator tool) seems safer than removing data. I pointed Peter here while we were in the ICU meeting, so hopefully he can resolve the issue separately.

Reply:
correct, 
- [ ] will do

Reply:
backed out rohyngya changes